navpy/core/navpy.py

```diff
@@ -41,6 +41,12 @@ def angle2dcm(rotAngle1, rotAngle2, rotAngle3, input_unit='rad',
     Returns
     --------
     C : {3x3} Direction Cosine Matrix
+        
```

Comment:
The ordering of the argument can be confusing and the example appears incorrect.  The ordering for `angle2dcm` arguments are described [here](https://navpy.readthedocs.io/en/latest/code_docs/coordinate_transformations.html#direction-cosine-matrices).  This proves the benefit of having an example!

In this case "yaw" happens first for ZYX ordering, so it should be the first argument.

Reply:
Resolved